import-service/serverless.ts

```diff
@@ -70,7 +78,46 @@ const serverlessConfiguration: Serverless = {
         }
       ]
 
+    },
+    catalogBatchProcess: {
+      handler: 'handler.catalogBatchProcess',
+      events: [
+        {
+          sqs: {
+            batchSize: 5,
+            arn: {
+              'Fn::GetAtt': ['SQSQueue', 'Arn']
+            }
+          }
+        }
+      ]
```

Comment:
This handler and all the services (sns, sqs) should be placed in products service instead of import service

Reply:
I am not agree. Since according to the homework there is no clear definition in which service we should implement SQS and SNS. So, to my point of view this is upon on developer.  Moreover, in task 6.2 Update the importFileParser lambda function (TASK 5) to send each CSV record into SQS. From this sentence i understood that we can implement it in the import-service.
Please, tell me if i am wrong?)

Reply:
Please see the video about SNS and SQS subscriptions on RS. On 1.29.25 you'll see Task 6 schema. "catalogBatchProcess" 
 Lambda is placed in Product Service.
As for SQS and SNS services:
- SNS. It is better to place it in Product Service since it is only used in catalogBatchProcess handler;
- SQS. It is up to you. It could be in import service and product service as well.
As for "catalogBatchProcess" handler:
Product Service is responsible for adding and retrieving data (products) from DB. So all the logic for adding and retrieving products should be implemented there (without "fetch" communication between services from one project)

Reply:
I understand I and I will fix it in the near future. Now I am a little bit busy, sorry:)